Packs/Salesforce/Integrations/SalesforceV2/SalesforceV2.py

```diff
@@ -0,0 +1,1041 @@
+
+    return get_case_command(client, {'oid': oid})
+
+
+def get_cases_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    properties = ['ID', 'CaseNumber', 'Subject', 'Description', 'CreatedDate',
+                  'ClosedDate', 'OwnerID', 'Priority', 'Origin', 'Status', 'Reason']
+    cases = client.queryObjects(properties, 'Case', '').get('records')
+    return objectToEntry(client, cases)
+
+
+def close_case_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    args['status'] = 'Closed'
+    return update_case_command(client, args)
+
+
+def delete_case_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    if args.get('oid', None) and args.get('caseNumber', None):
+        return_error('You must specify object ID or a Case Number')
+
+    if args.get('caseNumber', None):
+        condition = f"CaseNumber='{args.get('caseNumber')}'"
+        cases = client.queryObjects(['Id'], 'Case', condition).get('records')
+        if len(cases) > 0:
+            oid = cases[0].get('Id')
+        else:
+            return_error('Unable to update case -> Invalid Case Number provided')
+    else:
+        oid = args.get('oid')
+
+    return client.deleteObject(f'Case/{oid}')
+
+
+def push_comment_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    return client.pushComment(args.get('oid'), args.get('text'), args.get('linkUrl'))
+
+
+def push_comment_thread_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    results = client.pushCommentThread(args.get('id'), args.get('text'))
+
+    return results
+
+
+def search_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    searchRecords = client.sendRequestInSession(
+        'GET', 'search/?q=FIND+%7B' + args.get('pattern') + '%7D', '').get('searchRecords')
+    return searchToEntry(client, searchRecords)
+
+
+def list_case_files_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    fileInfo = client.getCaseFiles(args.get('caseoId', None), args.get('caseNumber', None)).get('fileInfo', [])
+
+    results = CommandResults(
+        outputs_prefix='Salesforce.Files',
+        outputs_key_field='id',
+        readable_output=tableToMarkdown(f"Case Files {args.get('caseoId', args.get('caseNumber', None))}", fileInfo),
+        outputs=fileInfo,
+        ignore_auto_extract=True)
+
+    return results
+
+
+def get_case_file_by_id_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    return client.getCaseFileById(args.get('caseNumber', None), args.get('caseFileId', None))
+
+
+def describe_sobject_field_command(client: Client, args: Dict[str, Any]):
+
+    found = False
+    response = client.sendRequestInSession('GET', 'sobjects/Case/describe/')
+
+    if args.get("field"):
+        fields = response.get('fields')
+        for field in fields:
+            if field['name'] == args.get("field"):
+                found = True
+                return field
```

Comment:
Please return the field in a CommandResults object.

Reply:
This comment is not relevant anymore because the Pack's support level has changed to - 'community'.